contributing.rst

```diff
@@ -0,0 +1,257 @@
+.. _contributing:
+
+===================
+Contributing to OSD
+===================
+
+OSD is an open-source project, and as such we welcome contributions from
+developers, engineers, system administrators, and users in general.
+
+There are many ways to contribute:
+
+Feedback
+========
+
+Feedback is very welcome! We are interested in hearing from you how! Email us
+@ `info@omnivector.solutions <mailto:info@omnivector.solutions>`_.
+
+In the case things aren't working as expected, please
+`file a bug <https://github.com/omnivector-solutions/slurm-charms/issues>`_.
+
+Reach out if you have any questions and/or suggestions.
+
+
+Documentation
+=============
+
+This documentation is built using `Sphinx <https://sphinx-doc.org/>`_ and
+`reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_,
+and is hosted in
+`GitHub <https://github.com/omnivector-solutions/osd-documentation>`_.
+
+General Recommendations
+-----------------------
+
+Use US English.
+
+Wrap the lines at 80 columns when writing documentation.
+
+Sections
+--------
+
+This projects uses the following convention for section headers, from the
+`rsTructuredText Primer <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections>`_
+document::
+
+   =================
+   This is a heading
+   =================
+
+   This is a section
+   =================
+
+   This is a subsection
+   --------------------
+
+   This is a subsubsection
+   ^^^^^^^^^^^^^^^^^^^^^^^
+
+   This is a subsubsubsection
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Naming conventions
+------------------
+
+Throughout the documentation and code (specially in logs), we stick to these
+naming conventions with regards to Slurm:
+
+``SLURM``
+   Schedmd Slurm project.
+
+``slurmXXXd``
+   The daemons for the Slurm component.
+
+   For example, ``slurmd`` refers to the daemon for the compute nodes, while
+   ``slurmctld`` refers to the daemon for the controller node.
+
+``Slurm``
+   Omnivector charms.
+
+
+Code
+====
+
+OSD source-code is hosted in GitHub repositories. All the charms are in
+`omnivector-solutions/slurm-charms <https://github.com/omnivector-solutions/slurm-charms>`_,
+and a common code shared among them, called ``slurm-ops-mananger``, is hosted at
+`omnivector-solutions/slurm-ops-manager <https://github.com/omnivector-solutions/slurm-ops-manager>`_.
+
+By contributing with code development, you agree the license used.
```

Comment:
```suggestion
By contributing to the slurm-charms project or affiliated code you are implicitly agreeing to the license used, MIT.
```

Reply:
We should also have a `Legal` section in the docs, what do you think?